File: app/src/main/java/com/example/alkewalletapp/fragmentos/SignupPage.java
package com.example.alkewalletapp.fragmentos;

import android.os.Bundle;

import androidx.annotation.NonNull;
import androidx.annotation.Nullable;
import androidx.fragment.app.Fragment;
import androidx.navigation.Navigation;

import android.view.LayoutInflater;
import android.view.View;
import android.view.ViewGroup;
import android.widget.Button;
import android.widget.TextView;

import com.example.alkewalletapp.R;

public class SignupPage extends Fragment {


    public SignupPage() {
        // Required empty public constructor
    }

    @Override
    public void onCreate(Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);
        if (getArguments() != null) {
        }
    }

    @Override
    public View onCreateView(LayoutInflater inflater, ViewGroup container,
                             Bundle savedInstanceState) {
        // Inflate the layout for this fragment
        return inflater.inflate(R.layout.fragment_signup_page, container, false);
    }

    @Override
    public void onViewCreated(@NonNull View view, @Nullable Bundle savedInstanceState) {
        super.onViewCreated(view, savedInstanceState);

        Button botonCrearCtaDos = view.findViewById(R.id.botoncrearcuenta2);
        TextView textYaTienesUna = view.findViewById(R.id.textyatienesuna);


        botonCrearCtaDos.setOnClickListener(new View.OnClickListener() {
            @Override
            public void onClick(View v) {
                Navigation.findNavController(v).navigate(R.id.action_signupPage_to_loginsignup);

            }
        });

        textYaTienesUna.setOnClickListener(new View.OnClickListener() {
            @Override
            public void onClick(View v) {
                Navigation.findNavController(v).navigate(R.id.action_signupPage_to_loginPage);
            }
        });

    }



}

File: app/src/main/java/com/example/alkewalletapp/fragmentos/SignupPage.kt
package com.example.alkewalletapp.fragmentos

import android.os.Bundle
import android.view.LayoutInflater
import android.view.View
import android.view.ViewGroup
import android.widget.Button
import android.widget.TextView
import androidx.fragment.app.Fragment
import androidx.navigation.Navigation.findNavController
import com.example.alkewalletapp.R

class SignupPage : Fragment() {
    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)
        if (arguments != null) {
        }
    }

    override fun onCreateView(
        inflater: LayoutInflater, container: ViewGroup?,
        savedInstanceState: Bundle?
    ): View? {
        // Inflate the layout for this fragment
        return inflater.inflate(R.layout.fragment_signup_page, container, false)
    }

    override fun onViewCreated(view: View, savedInstanceState: Bundle?) {
        super.onViewCreated(view, savedInstanceState)

        val botonCrearCtaDos = view.findViewById<Button>(R.id.botoncrearcuenta2)
        val textYaTienesUna = view.findViewById<TextView>(R.id.textyatienesuna)


        botonCrearCtaDos.setOnClickListener { v -> findNavController(v).navigate(R.id.action_signupPage_to_loginsignup) }

        textYaTienesUna.setOnClickListener { v -> findNavController(v).navigate(R.id.action_signupPage_to_loginPage) }
    }
}